source/includes/steps-create-index.yaml

```diff
@@ -82,11 +95,27 @@ content: |
       * - Wildcard projection (*New in MongoDB 4.2*)
         
         - If checked, support unknown or arbitrary fields
-          which match the specified projection in the index.
+          which match the specified projection in the index. You can
+          only specify a wildcard projection if your index field name is
+          ``$**``, meaning it covers all fields in the document.
+
+          .. example::
+
+             If your index field name is ``$**``, the following
+             wildcard projection document only includes
+             the values of the ``product_attributes.elements`` and
+             ``product_attributes.resistance`` fields in the index:
+
+             .. code-block:: javascript
+
+                {
+                  "product_attributes.elements" : 1,
+                  "product_attributes.resistance" : 1
+                }
```

Comment:
**issue:** The context sentence is dense. Consider breaking it up some to help with readability.

```suggestion
             Consider the following wildcard projection document:
             
             .. code-block:: javascript
                {
                  "product_attributes.elements" : 1,
                   "product_attributes.resistance" : 1
                }
             
             If your index field name is ``$**``, your index only
             includes the values of the fields in that projection.
```